inventory_management_system_api/models/catalogue_item.py

```diff
@@ -4,10 +4,23 @@
 
 from typing import Any, List, Optional
 
-from pydantic import BaseModel, ConfigDict, Field, HttpUrl, field_serializer, field_validator
-
-from inventory_management_system_api.models.custom_object_id_data_types import CustomObjectIdField, StringObjectIdField
-from inventory_management_system_api.models.mixins import CreatedModifiedTimeInMixin, CreatedModifiedTimeOutMixin
+from pydantic import (
+    BaseModel,
+    ConfigDict,
+    Field,
+    HttpUrl,
+    field_serializer,
+    field_validator,
+)
+
+from inventory_management_system_api.models.custom_object_id_data_types import (
+    CustomObjectIdField,
+    StringObjectIdField,
+)
+from inventory_management_system_api.models.mixins import (
+    CreatedModifiedTimeInMixin,
+    CreatedModifiedTimeOutMixin,
+)
```

Comment:
I think your vscode might be missing the `--line-length`, `120` options? Same in other files.